File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

a = parseInt(lines.shift())
b = parseInt(lines.shift())
q = parseInt(a/b)
r = a%b
        
if(r<0){
    if(q<0) q--
    else if(q>0) q++
    else{
        if(a-((q-1)*b)>=0){
            q--
        } else{
            q++
        }
    }
    r = a-(q*b)
}
        
console.log(q+" "+r)

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.py
ns = input().split()

a = int(ns[0])
b = int(ns[1])
q = a//b
r = a%b

if(r<0):
    if(a-((q-1)*b)>=0):
        q=q-1
    else:
        q=q+1

    r = a-(q*b)
   
print(q, r)